liqd_product/assets/scss/components/_userindicator.scss

```diff
@@ -0,0 +1,9 @@
+.userindicator__button {
```

Comment:
this rule is problematic, as it depends on the order of the rules in the css files which should be avoided. 
if it would be parsed prior to the .btn definition it would be less specific.
not sure how to fix it correctly though. but i would at least add a _comment to the css_. 

futhermore i think it could make sense to use ".userindicator__button.btn" or  ".userindicator__button.userindicator__button" which would be known to more specific but probably even worse in respect to css styles.

Reply:
Yes, good point! As all buttons on the main header are meant to look the same anyway, I renamed the button and used that. I am not sure if it should stay in the button scss, though or be moved to main_header